k8s/ithc/common/ccd/definition-store-api.yaml

```diff
@@ -19,9 +19,22 @@ spec:
   values:
     java:
       replicas: 2
+      keyVaults:
+        ccd:
+          secrets:
+            - definition-store-api-POSTGRES-USER-V11
```

Comment:
Does pr-974 tag has changes in bootstrap.yaml?

Reply:
Yes, it has